ngraph/core/src/type/element_type.cpp

```diff
@@ -72,8 +74,20 @@ static const element_types_map_t& get_type_info_map()
         {element::Type_t::u32, TypeInfo(32, false, false, false, "uint32_t", "u32")},
         {element::Type_t::u64, TypeInfo(64, false, false, false, "uint64_t", "u64")},
     };
-    return s_type_info_map;
-};
+
+    const ElementTypes::map_t& get_type_info_map() { return ElementTypes::map; };
+
+    const TypeInfo& get_type_info(element::Type_t type)
+    {
+        const auto& tim = get_type_info_map();
+        const auto& found = tim.find(type);
+        if (found == tim.end())
+        {
+            std::terminate(); // do not support other types
```

Comment:
@ilyachur, I'm not sure if we want such hard `terminate` here but don't see reason to call this functions with `type` which is not one of scope enumeration.

Reply:
Why cannot we throw an exception here?

Reply:
I have thought we have some set of type but after reconsideration `throw` might be better here. So I'll switch to `throw std::out_of_range`.